### ISSUE TITLE
Text features in Text widget should be implemented without TextStyle

### ISSUE THREAD

**Issue by Himanshuranjan30** [flutter/flutter#66002]
**Case:**
```
Text(
  'Hello, $_name! How are you?',
  textAlign: TextAlign.center,
  overflow: TextOverflow.ellipsis,
  style: TextStyle(fontWeight: FontWeight.bold,color:Colors.blue),
)

```
**Proposal:**
```
Text(
  'Hello, $_name! How are you?',
  textAlign: TextAlign.center,
  overflow: TextOverflow.ellipsis,
  fontWeight: FontWeight.bold),
  color: Colors.blue,
)
```

**Comment by pedromassangocode** [comment]
Hi @Himanshuranjan30 
Not sure if this is is a valid feature. Can you explain a little bit more what are the advantages of this proposal and the disadvantages of the actual approach?

**Comment by Himanshuranjan30** [comment]
Yes, TextStyle isn't used anywhere else except the Text Widget. So,Why Implement another class for styling texts when you can do it in the Text Widget class only right. This proposal can result in shorter and more clear implementation of text widgets.

**Comment by Himanshuranjan30** [comment]
Should I start working on this @pedromassangocode ?

**Comment by pedromassangocode** [comment]
Hi @Himanshuranjan30 
That would be great. I'm not the right person to decide if this is a valid proposal that should be added into Flutter.

This should be discussed first before any further action.

**Comment by Himanshuranjan30** [comment]
Sure,I will wait for the right person to get on this.
Thanks!

**Comment by PiN73** [comment]
Having `TextStyle` class makes it easy to separate style from actual text, define common style somewhere, use copyWith on style. Mixing it into `Text` would make all this more difficult or impossible
